FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_IPv6_Demo/IPv6_Multi_WinSim_demo/ReadMe.md

```diff
@@ -46,21 +46,46 @@ the location of source files:
 You can changes these directory to let the proyejct work with a different
 source tree.
 
-The Multiple Interface Visual studio demo showcases the Multiple Interfaces (or
-rather the multiple endpoints) functionality of the FreeRTOS+TCP
-IPv6/multi-interface library. The Windows Simulator environment doesn't actually
-have multiple interfaces which can be used by FreeRTOS and thus, this demo shows
+## Introduction
+
+The IPv6_Multi_WinSim Visual studio demo showcases the Multiple Interfaces (or
+rather the multiple endpoints) functionality of the FreeRTOS+TCP library devIPv6
+branch. The Windows Simulator environment doesn't actually have multiple
+interfaces which can be used by FreeRTOS and thus, this demo shows
 the use of different endpoints which will be resolved by the OS having multiple
 interfaces. It shows that the library will use different endpoints (IP-addresses)
 to connect to IP-addresses on different subnets (or using different netmasks).
-The instructions for the full Windows demo are still relevant though as they
-describe how to set up the WinPCap development environment, how to set the IP
-address, and other such items. Note that, as delivered, configUSE_DHCP is set to 0,
-so a static IP address is used. The instructions are provided on the following URL,
-see the "Hardware Setup" section:
+
+Note that, as delivered, configUSE_DHCP is set to 0, so a static IP address is used. 
```

Comment:
I think `configUSE_DHCP` is enabled and `configUSE_DHCPv6` is disabled (kept at default value).

Reply:
Yes that is correct. I will update.